docs/overview/office-add-in-code-samples.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Office Add-in code samples
+description: 'Office Add-in code samples'
```

Comment:
Can we have a more, well, descriptive description here?

Reply:
yes, added.